internal/command/e2etest/encryption_test.go

```diff
@@ -203,7 +215,10 @@ func TestEncryptionFlow(t *testing.T) {
 		apply().Failure().StderrContains("decryption failed for state")
 		requireEncryptedState()
 
-		applyPlan(encryptedPlan).Failure().StderrContains("decryption failed: cipher: message authentication failed")
+		tofuResultError := applyPlan(encryptedPlan).Failure()
+		SanitizeStderr(&tofuResultError)
+		tofuResultError.StderrContains("decryption failed: cipher: message authentication failed")
```

Comment:
`suggestion`
As a continuation of the comment on `SanitizeStderr`, once you have a new function created to handle the sanitisation of the stderr, would be better to call that func inside `StderrContains`. That way, all the testing structure in here will be able to make use of it.